apps/minifront/src/components/swap/dutch-auction/price.tsx

```diff
@@ -50,15 +43,6 @@ export const Price = () => {
           />
         </div>
       </div>
-
-      <div className='w-min'>
```

Comment:
`<Price />` no longer renders an asset selector, since that's now part of `<TokenSwapInput />`